tests/buyers/views/test_buyers.py

```diff
@@ -1528,6 +1528,91 @@ def test_links_to_sections_go_to_the_correct_pages_whether_they_be_sections_or_q
             assert section_4_link[0].get('href').strip() == \
                 '/buyers/frameworks/digital-outcomes-and-specialists/requirements/digital-specialists/1234/section-4'
 
+    def test_shows_correct_content_for_open_and_closed_dos_1_and_2_briefs(self, data_api_client):
+        framework_slugs = ["digital-outcomes-and-specialists", "digital-outcomes-and-specialists-2"]
+        framework_names = ["Digital Outcomes and Specialists", "Digital Outcomes and Specialists 2"]
+        sidebar_heading_content = ['Closing', 'Closed']
+        brief_statuses = ['live', 'closed']
+
+        for framework_slug, framework_name in zip(framework_slugs, framework_names):
+            with self.app.app_context():
+                self.login_as_buyer()
+
+                for heading, brief_status in zip(sidebar_heading_content, brief_statuses):
+                    data_api_client.get_framework.return_value = api_stubs.framework(
+                        slug=framework_slug,
+                        status='live',
+                        lots=[
+                            api_stubs.lot(slug='digital-specialists', allows_brief=True),
+                        ]
+                    )
+                    data_api_client.get_brief.return_value = api_stubs.brief(
+                        status=brief_status,
+                        framework_slug=framework_slug,
+                        framework_name=framework_name,
+                    )
+
+                    res = self.client.get(
+                        "/buyers/frameworks/{}/requirements/digital-specialists/1234".format(framework_slug)
+                    )
+                    assert res.status_code == 200
+
+                    document = html.fromstring(res.get_data(as_text=True))
+
+                    sidebar_headings = document.xpath('//*[@class="sidebar-heading"]/text()')
+                    sidebar_content = document.xpath('//*[@class="sidebar-content"]/text()')
+                    framework_name_content = document.xpath('//*[@class="framework-name"]/a/text()')[0]
+                    contract_link_text = document.xpath('//main[@id="content"]//ul/li/a/text()')[-1]
+
+                    assert sidebar_headings == ['Published', heading, 'Framework']
+                    assert sidebar_content == ['Tuesday 29 March 2016', 'Thursday 7 April 2016']
+                    assert framework_name_content == framework_name
+                    assert contract_link_text == "View the {} contract".format(framework_name)
```

Comment:
My thinking was keep content in one test, and link destinations in another. Something about the contracts may move but the link text should stay the same so separate tests? Not sure. 

I'll move it.